### PR TITLE
Lots of edits for Atom's package manager APM

### PR DESCRIPTION
**Pull request by thibmaek**
* Changed the keyword to apm (official package manager name for atom)
* Used a better icon which now is visible in light Alfred themes
* Added a screenshot for APM, this was forgotten somehow
* Made edits to the readme accordingly, correct a typo and increased version numbers.
* Gitignore for .DS_Store because it kept popping up

I didn't modify the workflow file since I couldn't figure out how and the icon kept reverting back to the old pixeled one 😕

### CODE REVIEW

**Comment by willfarrell** [comment]
Looks good, I can push the update in a moment

**Comment by thibmaek** [comment]
Thanks!

Damn seems I got some naming wrong.
The screenshot is named apm.png but in the README it's referenced as atom.png

**Comment by willfarrell** [comment]
done

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Package Managers ([Download v3.01](https://github.com/willfarrell/alfred-pkgman-workflow/releases/download/3.01/Package.Managers.alfredworkflow))
+# Package Managers ([Download v3.02](https://github.com/willfarrell/alfred-pkgman-workflow/releases/download/3.01/Package.Managers.alfredworkflow))
```
Comment (jeffbyrnes) [inline README.md]: I should point out that, while the link text changed here, the URL did not. Something to watch out for! I’ll update the contribution guidelines.

File: current-version.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"version": 3.01,
+	"version": 3.02,
 	"download_url": "https://github.com/willfarrell/alfred-pkgman-workflow/releases/download/3.01/Package.Managers.alfredworkflow",
```
Comment (jeffbyrnes) [inline current-version.json]: Same goes for here; gotta update those URLs. Again, my bad for not updating the contribution guidelines!

**Pull request merged**